completion/fish/task.fish

```diff
@@ -10,7 +10,7 @@ function __task_get_tasks --description "Prints all available tasks with their d
   end
 
   # Grab names and descriptions (if any) of the tasks
-  set -l output (echo $rawOutput | sed -e '1d; s/\* \(.*\):\s*\(.*\)\s*(aliases.*/\1\t\2/' -e 's/\* \(.*\):\s*\(.*\)/\1\t\2/'| string split0)
+  set -l output (echo $rawOutput | sed -e '1d; s/\* \(.*\):\s*\(.*\)\s*(\(aliases.*\))/\1\t\2\t\3/' -e 's/\* \(.*\):\s*\(.*\)/\1\t\2/'| string split0)
```

Comment:
Happy with this as a fix, but I think it would be good to move this complexity to Task itself sometime rather than relying on invoking `sed`. Looking at this expression gives me a headache 😆

Reply:
I totally agree with you, I would even argue that fish / bash / zsh does not support the same "feature" in the auto complete
As you said, we could move all complexity in our go code

Reply:
I've tested it in zsh, the behavior is the same, aliases are displayed but not "searchable"